src/app/header/header.component.scss

```diff
@@ -1,31 +1,30 @@
-header {
-  background-color: var(--ds-header-bg);
-}
+:host {
+  header {
+    background-color: var(--ds-header-bg);
+  }
 
-.navbar-brand img {
-  max-height: var(--ds-header-logo-height);
-  max-width: 100%;
-  @media screen and (max-width: map-get($grid-breakpoints, sm)) {
-    max-height: var(--ds-header-logo-height-xs);
+  .navbar-brand img {
+    max-height: var(--ds-header-logo-height);
+    max-width: 100%;
   }
-}
 
-.navbar-toggler .navbar-toggler-icon {
-  background-image: none !important;
-  line-height: 1.5;
-}
+  .navbar-toggler .navbar-toggler-icon {
+    background-image: none !important;
```

Comment:
I assume the `!important` is here to counter one from bootstrap, so that would be a legitimate case to use it (as that would be the only way to override another `!important`)

Reply:
Yes, but this can actually be removed as I only needed the width and height of the `.navbar-toggler-icon` Bootstrap class.

Since these values are required only in the _base_ theme I'm going to hardcode them in the CSS files, as defining a CSS variable that is not used in the _dspace_ theme may be misleading